gradle/testing.gradle

```diff
@@ -18,31 +18,57 @@
  * limitations under the License.
  */
 
-if (hasProperty('testEnableCoverage')) {
-    apply plugin: 'jacoco'
-    tasks.withType(Test) { task ->
-        finalizedBy "jacoco${task.name.capitalize()}Report"
+apply plugin: 'jacoco'
```

Comment:
The removal of this `coverage.gradle` means we are no longer aggregating the information.
I *think* consumers can gather the files with something like:
```
*/.out/reports/jacoco/*/*.xml
```

I tried to get this working with `jacoco-report-aggregation`, but I couldn't get that to work with the subprojects that we have.
I don't know how to get it working with sonarcloud, but that doesn't work now either.